samples/snippets/cloud-client/src/test/java/privateca/SnippetsIT.java

```diff
@@ -138,73 +127,40 @@ public static void cleanUp() throws InterruptedException, ExecutionException, IO
-    try (KeyManagementServiceClient client = KeyManagementServiceClient.create()) {
-      // Build the parent name from the project and location.
-      LocationName locationName = LocationName.of(PROJECT_ID, KMS_LOCATION);
-
-      // Build the key ring to create.
-      KeyRing keyRing = KeyRing.newBuilder().setName(locationName.toString()).build();
-
-      // Create the key ring.
-      KeyRing createdKeyRing =
-          client.createKeyRing(
-              CreateKeyRingRequest.newBuilder()
-                  .setParent(locationName.toString())
-                  .setKeyRing(keyRing)
-                  .setKeyRingId(KEY_RING_ID)
-                  .build());
-      System.out.printf("Created key ring: %s%n", createdKeyRing.getName());
-    }
-  }
-
-  // Create a new asymmetric key for the purpose of signing and verifying data.
-  public static void createAsymmetricSignKey() throws IOException {
-    // Initialize client that will be used to send requests. This client only
-    // needs to be created once, and can be reused for multiple requests. After
-    // completing all of your requests, call the "close" method on the client to
-    // safely clean up any remaining background resources.
-    try (KeyManagementServiceClient client = KeyManagementServiceClient.create()) {
-      // Build the parent name from the project, location, and key ring.
-      KeyRingName keyRingName = KeyRingName.of(PROJECT_ID, KMS_LOCATION, KEY_RING_ID);
-
-      // Build the asymmetric key to create.
-      CryptoKey key =
-          CryptoKey.newBuilder()
-              .setPurpose(CryptoKeyPurpose.ASYMMETRIC_SIGN)
-              .setVersionTemplate(
-                  CryptoKeyVersionTemplate.newBuilder()
-                      .setAlgorithm(CryptoKeyVersionAlgorithm.RSA_SIGN_PKCS1_2048_SHA256))
-              .build();
-
-      // Create the key.
-      CryptoKey createdKey = client.createCryptoKey(keyRingName, KEY_ID, key);
-      System.out.printf("Created asymmetric key: %s%n", createdKey.getName());
-    }
-  }
-
-  public static void cleanupCertificateSignKey() throws IOException, InterruptedException {
-    try (KeyManagementServiceClient client = KeyManagementServiceClient.create()) {
-      CryptoKeyVersionName cryptoKeyVersionName =
-          CryptoKeyVersionName.of(PROJECT_ID, KMS_LOCATION, KEY_RING_ID, KEY_ID, VERSION_ID);
-      // Destroy the crypto key version.
-      CryptoKeyVersion cryptoKeyVersion = client.destroyCryptoKeyVersion(cryptoKeyVersionName);
-      sleep(5);
-      // If the response has destroy time, then the version is successfully destroyed.
-      Assert.assertTrue(cryptoKeyVersion.hasDestroyTime());
-    }
-  }
-
   // Wait for the specified amount of time.
   public static void sleep(int seconds) throws InterruptedException {
     TimeUnit.SECONDS.sleep(seconds);
   }
 
+  // Create an asymmetric key pair to be used in certificate signing.
+  public static ByteString createPublicKey()
```

Comment:
suggestion: change this method to something like `createPemEncodedKeyPair()`, and return both the private and public keys.

Reply:
Thank you. That's a great way to demonstrate how to generate both public and private keys. 
Will modify it!